models/reviews.models.js

```diff
@@ -13,16 +13,20 @@ exports.selectReviewById = (id) => {
 };
 
 exports.updateVotesById = (id, votes) => {
-  return db
-    .query(
-      `UPDATE reviews SET votes = $1 + votes WHERE review_id = $2 RETURNING *;`,
-      [votes, id]
-    )
-    .then(({ rows }) => {
-      if (rows.length > 0) {
-        return rows[0];
-      } else {
-        return Promise.reject({ status: 404, msg: "Not Found" });
-      }
-    });
+  if (typeof votes !== "number") {
```

Comment:
16-17 - this works fine, but if you wanted to you could not have it.  postgres will throw an error if it receives an incorrect data type, and you can pass that error to your custom error handler.  But it is fine if you want to keep this.